src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,15 +14,388 @@ tree_item_index: 1100
+
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+  {
+    "Sid": "VisualEditor0",
+    "Effect": "Allow",
+    "Action": [
+    "s3:PutObject",
+    "s3:GetObject",
+        "s3:AbortMultipartUpload",
+        "s3:DeleteObject",
+        "s3:ListMultipartUploadParts"
+      ],
+      "Resource": "arn:aws:s3:::bucket-name/*"
+    },
+    {
+      "Sid": "VisualEditor1",
+      "Effect": "Allow",
+      "Action": [
+        "s3:ListBucketMultipartUploads",
+        "s3:ListBucket",
+        "s3:GetBucketLocation"
+      ],
+      "Resource": "arn:aws:s3:::bucket-name"
+    }
+  ]
+}
+```
+
+#### Nuxeo Configuration
+
+##### Transient Store on AWS
+
+To use S3 direct upload with Nuxeo, you will need to add another policy to a new role **for example**:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListAllMyBuckets"
+            ],
+            "Resource": "arn:aws:s3:::*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetBucketLocation",
+                "s3:AbortMultipartUpload",
+                "s3:ListMultipartUploadParts",
+                "s3:ListBucketMultipartUploads"
+            ],
+            "Resource": "arn:aws:s3:::yourbucketname"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:DeleteObject",
+                "s3:AbortMultipartUpload",
+                "s3:ListMultipartUploadParts",
+                "s3:ListBucketMultipartUploads"
+            ],
+            "Resource": "arn:aws:s3:::yourbucketname/*"
+        }
+    ]
+}
+```
+
+Make sure that all configuration values are set in the `nuxeo.conf`:
+
+```
+nuxeo.s3storage.transient.bucket=
+nuxeo.aws.accessKeyId=
+nuxeo.aws.secretKey=
+nuxeo.aws.region=
+nuxeo.s3storage.bucket=
+```
+{{#> callout type='note'}}
+
+Since the work in [NXP-29548](https://jira.nuxeo.com/browse/NXP-29548) 'nuxeo.s3storage.useDirectUpload=true' and 'nuxeo.s3storage.transient.roleArn=' (documented as part of the previous version configuration) are no longer necessary **for the Nuxeo Aspera Connector** and can be removed, **if you only use these values for the Nuxeo Aspera Connector, and not for any other reason**.  See upgrade notes.
+{{/callout}}
+
+#### Aspera Nuxeo Configuration
+
+Add the aspera access keys in `nuxeo.conf`:
+
+**Example**:
+
+```
+## is the same for the 2 nodes
+aspera.node.url=https://ats-aws-us-east-1.aspera.io
+aspera.node.port=443
+
+### ACCESS KEY ON UPLOAD NODE LINKED TO TS S3 BUCKET
+aspera.access.key.id=
+### SECRET KEY ON UPLOAD NODE LINKED TO TS S3 BUCKET
+aspera.access.key.secret=
+
+### ACCESS KEY ON DOWNLOAD NODE LINKED TO TMAIN S3 BUCKET
+aspera.download.access.key.id=
+### SECRET KEY ON DOWNLOAD NODE LINKED TO MAIN S3 BUCKET
+aspera.download.access.key.secret=
```

Comment:
I would suggest that we could put values everywhere. Fake ones, or course.
Or simply don't put any value and leave everything empty as above for `s3` configs. 

Mixing both approaches doesn't make sense IMHO